nengo/tests/test_pytest.py

```diff
@@ -1,8 +1,35 @@
 import nengo.utils.numpy as npext
 from nengo.conftest import TestConfig
 
+pytest_plugins = ["pytester"]
+
 
 def test_seed_fixture(seed):
     """The seed should be the same on all machines"""
     i = (seed - TestConfig.test_seed) % npext.maxint
     assert i == 1832276344
+
+
+def test_unsupported(testdir):
+    testdir.makeconftest(
+        """
+        import nengo.conftest
+
+        class MockSimulator(object):
+            unsupported = [('*', 'mock simulator')]
+
+        nengo.conftest.TestConfig.Simulator = MockSimulator
+        """)
+    outcomes = testdir.runpytest(
+        "-p", "nengo.tests.options", "--pyargs", "nengo",
+    ).parseoutcomes()
+    assert outcomes["skipped"] > 350
+    assert outcomes["deselected"] > 700
+    assert "passed" not in outcomes
+    assert "failed" not in outcomes
+
+    outcomes = testdir.runpytest(
+        "-p", "nengo.tests.options", "--pyargs", "nengo", "--unsupported",
+    ).parseoutcomes()
+    assert "passed" not in outcomes
```

Comment:
Should we be checking that `outcomes['xpassed'] + outsomes['xfailed'] > 350` or something like that here?

Reply:
Huh... so yesterday this part of the test wasn't working correctly for me due to some weirdness with Matplotlib, but I just ran it now and it seems to be working correctly, so yeah we should.

Reply:
Looks good!